terraform/modules/cloud/aws/deploy/fleet/templates/user_data.bash

```diff
@@ -26,13 +32,11 @@ local
 EOF
 
 ansible-playbook \
-    -i inventory/vault.yml \
```

Comment:
I see this file had already been deleted so I expect no functional change from this deletion.